src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,56 @@
+import classnames from 'classnames';
+import React, { FC, useState } from 'react';
+
+import styles from './Checkbox.module.css';
+
+import { ICheckboxProps } from './ICheckbox.d';
+
+const Checkbox: FC<ICheckboxProps> = ({
+  id,
+  variant = 'primary',
+  label,
+  indeterminate,
+  disabled,
+  ...props
+}) => {
+  const [checked, setChecked] = useState(false);
```

Comment:
this should be dumb component, it should take the `checked` value as a prop and based on this value it will do whatever we want